src/mame/casio/cz1.cpp

```diff
@@ -910,7 +910,7 @@ void cz1_state::cz1(machine_config &config)
 	m_mcu->t0_in_cb().set(FUNC(cz1_state::sync49_r));
 	m_mcu->t1_in_cb().set([this] () { return BIT(m_main_port[2], 7); });
 
-	MSM6200(config, "kbd").irq_cb().set_inputline(m_mcu, MCS48_INPUT_IRQ);
+	MSM6200(config, "kbd", 2'470'000).irq_cb().set_inputline(m_mcu, MCS48_INPUT_IRQ);
```

Comment:
Where’s this frequency coming from?

Reply:
There's an oscillator on the main board that goes out to the keyboard PCB (it's hard to make out in the schematic because of the poor scan quality, but the parts list makes it more clear)
![image](https://github.com/user-attachments/assets/32e2b241-6f15-43ab-838e-4b368b7fc9ef)
![image](https://github.com/user-attachments/assets/cb4b1d81-a0a9-404e-bb80-c8783dd43f89)

Reply:
You’re supposed to use `_XTAL` markup for those.  I’ll fix it.

Reply:
Yeah, if it’s a crystal/resonator, use `_XTAL` markup.  If it’s an RC/LC circuit or something, just put in the frequency.

Reply:
Yes xtal.cpp includes ceramic oscillators.

Reply:
Good to know, thanks.